### PR TITLE
Add tests

### LINKED ISSUE #50: Test Coverage: RecipeParserUtilTest, Undo/Redo and more
Reference #20 
RecipeParserUtilTest incomplete, does not test for ingredient-related parsing.

### PR DESCRIPTION
**Pull request by Rahul0506**

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/RedoCommandIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+        assertCommandSuccess(new RedoCommand(), model, expectedResult, model);
+    }
+
+    @Test
+    public void execute_redoAfterOneCommand_failure() {
+        try {
+            new AddItemCommand(new ItemPrecursorBuilder().build()).execute(model);
+        } catch (Exception ignored) {
+            // ignored
```
Comment (aqcd) [inline src/test/java/seedu/address/logic/commands/RedoCommandIntegrationTest.java]: document why it is ignored, or add logger (same for other similar instances)

File: src/test/java/seedu/address/model/item/NameMatchesKeywordsPredicateTest.java [hunk 2]
```diff
@@ -60,7 +60,7 @@ public void test_nameContainsKeywords_returnsTrue() {
     }
 
     @Test
-    public void test_nameDoesNotContainKeywords_returnsFalse() {
+    public void test_nameContainsKeywords_returnsFalse() {
```
Comment (aqcd) [inline src/test/java/seedu/address/model/item/NameMatchesKeywordsPredicateTest.java]: why this change?

Comment (Rahul0506) [reply src/test/java/seedu/address/model/item/NameMatchesKeywordsPredicateTest.java]: huh, yeah that was dumb

**Review by aqcd — APPROVED** [review]
LGTM
